workspaces/confluence/plugins/search-backend-module-confluence-collator/README.md

```diff
@@ -61,9 +61,10 @@ There is some configuration that needs to be setup to use this action, these are
 confluence:
   baseUrl: 'http://confluence.example.com'
   auth:
+    type: 'bearer' # can also be 'basic' or 'userpass'
     token: '${CONFLUENCE_TOKEN}'
-  spaces: [] # Warning, it is highly recommended to safely list the spaces that you want to index, either all documents will be indexed.
-  query: '' # If your spaces contain documents you don't want to index, you can use a CQL query to more precisely select them. This is combined with the spaces parameter above
+  spaces: [] # It is highly recommended to safely list the spaces that you want to index, otherwise all spaces will be indexed.
+  query: '' # If your spaces contain documents you don't want to index, you can use a CQL query to more precisely select them. This is combined with the spaces parameter above. Documentation about CQL can be found here `https://developer.atlassian.com/server/confluence/advanced-searching-using-cql`
```

Comment:
Can we move this out so the link is clickable instead of having to copy and paste it, please?